acl/src/main/java/org/apache/rocketmq/acl/plain/PlainAccessResource.java

```diff
@@ -289,6 +425,42 @@ public static boolean isRetryTopic(String topic) {
         return null != topic && topic.startsWith(MixAll.RETRY_GROUP_TOPIC_PREFIX);
     }
 
+    public static boolean isContainNamespace(String resource) {
```

Comment:
Use methods provided in `NamespaceUtil` if possible, the logic of processing namespace should not be included in ACL module.